src/manager/manager.c

```diff
@@ -313,6 +312,12 @@ bool manager_parse_config(Manager *manager, const char *configfile) {
                 return false;
         }
 
+        result = cfg_manager_def_conf(manager->config);
+        if (result != 0) {
+                fprintf(stderr, "Error setting default config values in manager instance");
```

Comment:
```suggestion
                fprintf(stderr, "Failed to set default settings for manager: %s", strerror(-result));
```
I think these error logs should be the same for agent and manager.